python/ray/train/lightning/lightning_trainer.py

```diff
@@ -427,6 +447,17 @@ def _check_checkpoint_configs(
                 "through `LightningConfigBuilder.checkpointing()`."
             )
 
+        # Auto-fill the AIR CheckpointConfig if the user didn't specify it.
+        if air_ckpt_config == CheckpointConfig():
```

Comment:
One thing to be aware of is that if the user does explicitly pass in a default `CheckpointConfig`, then this will also get overridden. Is that expected?